src/txverificationqueue.h

```diff
@@ -0,0 +1,26 @@
+class CTxVerificationQueueEntry
+{
+private:
+    CTransaction tx;
```

Comment:
some data member is not actually used and can be removed as well as mutator related methods